brave/brave-core
# Adds notes to the panel and site banner

Resolves https://github.com/brave/brave-browser/issues/2697

UI implementation: https://github.com/brave/brave-ui/pull/331

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- enable rewards
- visit non verified publisher
- open panel and site banner (make sure that is the same as in https://github.com/brave/brave-browser/issues/2697)
- open panel and click change settings
- change setting to off
- go back to site
- open panel
- make sure that change setting is not there anymore

## Reviewer Checklist:
